src/workspacer/Workspace/WorkspaceManager.cs

```diff
@@ -417,7 +417,8 @@ public void UpdateWindow(IWindow window, WindowUpdateType type)
                         }
                     }
 
-                    if (type == WindowUpdateType.Move)
+                    // Only change the current window if we are actively moving it.
+                    if (type == WindowUpdateType.Move && window.IsMouseMoving)
```

Comment:
This seems like a unrelated change... Was this actually required, or is it just something else you discovered and decided to fix at the same? :smile: